### ISSUE TITLE
Increase cell input voltage to 9V

### ISSUE THREAD

**Issue by napowderly** [atopile/cell-sim#1]
Unable to output more than 4.3V, I believe this is due to minimum dropout across the buck and LDO regulators.
Simple solution is to increase the input to 9V, limitation is the LDO is only capable to 5.5V input, might need to switch for something that could in worst case survive 9V input.

**Comment by mawildoer** [comment]
Isn't there a buck in-between? If so, then limiting the buck's output voltage could also solve it

**Comment by napowderly** [comment]
Yeah there is a buck, could potentially do that, its slightly complicated by the DAC control of the feedback node.
